src/Utils/Events.ts

```diff
@@ -1,7 +1,15 @@
 import EventEmitter from "events"
 
```

Comment:
Where does this events api come from?

Reply:
It's from Node's standard library and pollyfilled (via browserify/webpack) for browsers.